File: SampSharpGameMode/Story/Objects/Laptop.cs
using SampSharp.GameMode;
using SampSharp.Streamer;
using SampSharp.Streamer.World;
using System;
using System.Collections.Generic;
using System.Text;
using System.Threading.Tasks;

namespace SampSharp.SyntheticGameMode.Story.Objects
{
    public class Laptop : SampStoryObjectBase
    {
        public enum eLaptopModel
        {
            eClosed = 19894,
            eOpen = 19893
        }

        public override string Description
        {
            get => this.ModelId switch
            {
                19894 => " closed laptop",
                19893 => " open laptop",
                _ => " laptop ",
            };
            set { }
        }
    }
}

File: story/Objects/Laptop.lua
Laptop = class(SampStoryObjectBase, function(o, params)
    params.description = "laptop"
    SampStoryObjectBase.init(o, params)
    o:updateDescription()
end
)

function Laptop:updateDescription()
    if self.modelid == Laptop.eModel.Closed then
        self.Description = "closed lid laptop"
    elseif self.modelid == Laptop.eModel.Open then
        self.Description = "open lid laptop"
    end
    return self.Description
end

Laptop.eModel = 
{
    Closed = 7188,
    Open = 7187
}